app/controllers/requesters_controller.rb

```diff
@@ -0,0 +1,74 @@
+class RequestersController < ApplicationController
+  before_action :set_requester, only: [:show, :edit, :update, :destroy]
+
+  # GET /requesters
+  # GET /requesters.json
+  def index
+    @requesters = Requester.all
+  end
+
+  # GET /requesters/1
+  # GET /requesters/1.json
+  def show
+  end
+
+  # GET /requesters/new
+  def new
+    @requester = Requester.new
+  end
+
+  # GET /requesters/1/edit
+  def edit
+  end
+
+  # POST /requesters
+  # POST /requesters.json
+  def create
+    @requester = Requester.new(requester_params)
+
+    respond_to do |format|
+      if @requester.save
+        format.html { redirect_to @requester, notice: 'Requester was successfully created.' }
+        format.json { render :show, status: :created, location: @requester }
+      else
+        format.html { render :new }
+        format.json { render json: @requester.errors, status: :unprocessable_entity }
```

Comment:
I really like that you're including the json handling. Do you think including a test for this error state is appropriate yet?

Reply:
Ah yes! Thanks rails scaffolding. :)

We were thinking we could merge this in with just the model tests for now since it will be helpful to have the model in for other work. However I think it is a good idea to include a test for the error state eventually!

Reply:
@mirandawang Can you include the scaffolding command you used? I think it would be good for us to standardize